pkg/minikube/registry/drvs/podman/podman.go

```diff
@@ -38,13 +38,19 @@ import (
 	"k8s.io/minikube/pkg/minikube/registry"
 )
 
+var docURL = "https://minikube.sigs.k8s.io/docs/drivers/podman/"
+
 // minReqPodmanVer is required the minimum version of podman to be installed for podman driver.
-var minReqPodmanVer = semver.Version{Major: 1, Minor: 7, Patch: 0}
+var minReqPodmanVer = semver.Version{Major: 2, Minor: 1, Patch: 0}
 
 func init() {
-	priority := registry.Experimental
+	priority := registry.Default
```

Comment:
how about we add Funcitonal Test on Github Action for Podman before we bump up the priority?

Reply:
Sounds like a good suggestion, please add it to the issue

Reply:
Still works fine with Podman 2.2.1, though. So nothing to change in this PR, just because 3.0.0 was buggy

Reply:
Now 3.0.1 is out as well, help would be needed for adding a functional test